src/core/highlight/cell.selector.js

```diff
@@ -7,7 +7,7 @@ export default (model, markup) => {
 		const rows = model.view().rows;
 		
 		for (let item of items) {
-			const index = rows.indexOf(item);
+			const index = rows.indexOf(item) * 2;
```

Comment:
Yeah) we have two tr for each row one for row data itself and second one for details